src/kOS/Suffixed/Timespan.cs

```diff
@@ -42,16 +42,16 @@ private void InitializeSuffixes()
 
         private int CalculateYear()
         {
-            if (kerbinTimeSetting)
+            if (GameSettings.KERBIN_TIME)
             {
                 return (int)Math.Floor(span / SECONDS_IN_KERBIN_YEAR) + 1;
             }
             return (int)Math.Floor(span / SECONDS_IN_EARTH_YEAR) + 1;
         }
 
-        private int SecondsPerDay { get { return kerbinTimeSetting ? SECONDS_IN_KERBIN_DAY : SECONDS_IN_EARTH_DAY; } }
-        private int SecondsPerHour { get { return kerbinTimeSetting ? SECONDS_IN_KERBIN_HOUR : SECONDS_IN_EARTH_HOUR; } }
-        private int SecongsPerYear { get { return kerbinTimeSetting ? SECONDS_IN_KERBIN_YEAR : SECONDS_IN_EARTH_YEAR; } }
+        private int SecondsPerDay { get { return GameSettings.KERBIN_TIME ? SECONDS_IN_KERBIN_DAY : SECONDS_IN_EARTH_DAY; } }
+        private int SecondsPerHour { get { return GameSettings.KERBIN_TIME ? SECONDS_IN_KERBIN_HOUR : SECONDS_IN_EARTH_HOUR; } }
+        private int SecongsPerYear { get { return GameSettings.KERBIN_TIME ? SECONDS_IN_KERBIN_YEAR : SECONDS_IN_EARTH_YEAR; } }
```

Comment:
The above change wasn't *strictly* necessary anymore after I decided not to make ``kuniverse:hoursperday`` settable after all.  However, just to be more tolerant of other mods that might change the setting like we were about to do, I left this in place.